great_expectations_provider/__init__.py

```diff
@@ -9,4 +9,11 @@ def get_provider_info():
         "name": "Great Expectations Provider",
         "description": "An Apache Airflow provider for Great Expectations.",
         "versions": [__version__],
```

Comment:
```suggestion
        "versions": [__version__],
        "hooks": [
            {"integration-name": "GX Cloud", "python-modules": ["great_expectations_provider.hooks.gx_cloud"]}
        ],
```
would be nice to also register the hook here similar to https://github.com/apache/airflow/blob/a54d1d9a2eb2517f4ba6501a06972ac9a5fe4c38/providers/cohere/src/airflow/providers/cohere/get_provider_info.py#L40C8-L42C11